### ISSUE TITLE
Handle locked route when no redirect parameter

### ISSUE THREAD

**Issue by jongacnik** [johannschopplich/kirby-locked-pages#2]
**Current Behavior:**
When navigating to `/locked` without any query parameters, an error is thrown since the `LockedPages::find` method expects a `Page` object, which it infers from the `?redirect=page-uri` value.

**Expected/Possible Behavior**
Render the Error page when trying to directly access `/locked`. This can be done by adding a simple check at the top of the [controller](https://github.com/johannschopplich/kirby-locked-pages/blob/master/controllers/locked-pages-login.php), such as:

```php
return function ($kirby) {
    $id = get('redirect');
    $targetPage = page($id);

    if (!$id || !$targetPage) {
        go('error');
    }
```

**Comment by johannschopplich** [comment]
Thanks, great catch. For your solution I'd prefer `go(site()->errorPage())` just in case it's slug differs from called `error`.

For semantics and performance I will return earlier, directly before the `new Page()` is initiated:

```php
if (!get('redirect')) {
  return false; // Error page
}

```

**Comment by johannschopplich** [comment]
Fixed in v1.0.4. Thanks again for the issue.
